tests/test_recipes.py

```diff
@@ -41,6 +41,14 @@ def test_import_seq_from_recipe():
         pytest.fail("{} raised".format(ImportError.__name__))
 
 
+def test_import_recipes():
+    """Test imports works both for full import paths and for
+    `app_name.receipe_name` strings."""
+    assert baker.prepare_recipe("generic.dog"), baker.prepare_recipe(
+        "tests.generic.dog"
```

Comment:
```suggestion
        "tests.generic.Dog"
```

(it works but just for the sake of convention)